sklearn/cluster/_feature_agglomeration.py

```diff
@@ -62,43 +62,30 @@ def transform(self, X):
             nX = np.array(nX).T
         return nX
 
-    def inverse_transform(self, Xt=None, Xred=None):
+    def inverse_transform(self, X=None, Xt=None, Xred=None):
         """
         Inverse the transformation and return a vector of size `n_features`.
 
         Parameters
         ----------
-        Xt : array-like of shape (n_samples, n_clusters) or (n_clusters,)
+        X : array-like of shape (n_samples, n_clusters) or (n_clusters,)
             The values to be assigned to each cluster of samples.
 
-        Xred : deprecated
-            Use `Xt` instead.
+        Xt : array-like of shape (n_samples, n_clusters) or (n_clusters,)
+            The values to be assigned to each cluster of samples.
 
-            .. deprecated:: 1.3
+            .. deprecated:: 1.5
+                `Xt` was deprecated in 1.5 and will be removed in 1.7. Use `X` instead.
 
         Returns
         -------
         X : ndarray of shape (n_samples, n_features) or (n_features,)
             A vector of size `n_samples` with the values of `Xred` assigned to
             each of the cluster of samples.
         """
-        if Xt is None and Xred is None:
-            raise TypeError("Missing required positional argument: Xt")
-
-        if Xred is not None and Xt is not None:
-            raise ValueError("Please provide only `Xt`, and not `Xred`.")
-
-        if Xred is not None:
-            warnings.warn(
-                (
-                    "Input argument `Xred` was renamed to `Xt` in v1.3 and will be"
-                    " removed in v1.5."
-                ),
-                FutureWarning,
-            )
-            Xt = Xred
```

Comment:
I've remove these based on the logic. If there is an additional removal step or it should be separate, let me know